Doc/faq/gui.rst

```diff
@@ -64,7 +64,7 @@ I can't get key bindings to work in Tkinter: why?
 -------------------------------------------------
 
 An often-heard complaint is that event handlers bound to events with the
-:meth:`bind` method don't get handled even when the appropriate key is pressed.
+:meth:`!bind` method don't get handled even when the appropriate key is pressed.
```

Comment:
Perhaps we could link to https://docs.python.org/3/library/tkinter.html#bindings-and-events here.